src/blockchain/shelley/block.js

```diff
@@ -112,11 +120,16 @@ export default class ShelleyBlock implements Block {
       if (fragment.is_transaction()) console.log(`#${index} = TRANSACTION`)
       if (fragment.is_owner_stake_delegation()) console.log(`#${index} = OWNER STAKE DELEG`)
       if (fragment.is_stake_delegation()) console.log(`#${index} = STAKE DELEG`)
-      // if (fragment.is_pool_registration()) console.log('#' + index + ' = POOL REG')
```

Comment:
Was this too spammy? Why are the other ones still there? I mean ideally we'd have them as using the `Logger` and I had just put them in to see if it was parsing the certs. I'm just confused why that one was commented out then removed while the others weren't though.